src/js/_enqueues/admin/site-icon.js

```diff
@@ -141,32 +143,52 @@
 	 * @param {array} attributes The attributes for the attachment.
 	 */
 	function switchToUpdate( attributes ) {
-		var i18nAltString;
-
-			if ( attributes.alt ) {
-				i18nAltString = wp.i18n.sprintf(
-					/* translators: %s: The selected image alt text. */
-					wp.i18n.__( 'Current image: %s' ),
-					attributes.alt
-				);
-			} else {
-				i18nAltString = wp.i18n.sprintf(
-					/* Translators: %s: The selected image filename. */
-					wp.i18n.__( 'The current image has no alternative text. The file name is: %s' ),
-					attributes.filename
-				);
-			}
+		var i18nAltString,
+		i18nAppAlternativeString,
+		i18nBrowserAlternativeString;
+
+		if ( attributes.alt ) {
+			i18nAltString = wp.i18n.sprintf(
+				/* translators: %s: The selected image alt text. */
+				wp.i18n.__( 'Current image: %s' ),
+				attributes.alt
+			);
+		} else {
+			i18nAltString = wp.i18n.sprintf(
+				/* Translators: %s: The selected image filename. */
+				wp.i18n.__( 'The current image has no alternative text. The file name is: %s' ),
+				attributes.filename
+			);
+		}
+
+		i18nAppAlternativeString = wp.i18n.sprintf(
+			/* Translators: 1: Prefix used for alternative text of the preview of the site icon. 2: Alternative text of the site icon image, */
+			wp.i18n.__( '%1$s: %2$s' ),
```

Comment:
This would solve the a11y need to clarify the alt text. However, I'm not sure concatenation is ideal from an l10n perspective and for translators. Ideally, these should be full sentences, avoiding concatenation.

Reply:
It's also best to use the same strings as are used in PHP so that they are translated the same. Additionally, while in English (and some other languages), it makes sense to put the alt text after the prefix, this isn't something that can be assumed for all languages (Some could move the `$s` elsewhere in the sentence).

Reply:
Placeholders could be shuffled during translations, i.e. `%$2s (%1$s)`, thats why I created at translatable concatenation  string, however, I added four full strings that can be translated independently now.